addons/hw_posbox_homepage/controllers/main.py

```diff
@@ -41,15 +42,11 @@
 class IoTboxHomepage(odoo.addons.web.controllers.main.Home):
 
     def get_ip_iotbox(self):
-        interfaces = ni.interfaces()
-        for iface_id in interfaces:
-            iface_obj = ni.ifaddresses(iface_id)
-            ifconfigs = iface_obj.get(ni.AF_INET, [])
-            for conf in ifconfigs:
-                if conf.get('addr') and conf.get('addr') != '127.0.0.1':
-                    ips = conf.get('addr')
-                    break
-        return ips
+        try:
+            return ni.ifaddresses('eth0')[ni.AF_INET][0]['addr']
+        except:
+            return ni.ifaddresses('wlan0')[ni.AF_INET][0]['addr']
```

Comment:
don't you have a `get_ip` helper? why not use it here?
don't shorten netifaces to ni 'cause i didn't get it at first